packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -644,9 +644,9 @@ export const mutation = mutationType({
       },
     })
 
-    t.field('refreshOrganizations', {
+    t.field('refreshCloudViewer', {
```

Comment:
Renaming this to make it clear this invalidates more than just the organizations data